mass_flask_core/tests/test_request_dispatch.py

```diff
@@ -0,0 +1,19 @@
+from mixer.backend.mongoengine import mixer
+from mongoengine import DoesNotExist
+
+from mass_flask_core.models import AnalysisSystem, AnalysisRequest, IPSample, URISample
+from mass_flask_core.tests import FlaskTestCase
+
+
+class RequestDispatchTestCase(FlaskTestCase):
+    def test_are_requests_correct_after_sample_creation(self):
```

Comment:
There should exist a similar test case where first the sample is created and then the analysis system.